tooling/FST-1030-FSharp.Core nuget package netstandard 2.0 only.md

```diff
@@ -0,0 +1,59 @@
+
+This will work well, because the Windows desktop and netstandard FSharp.Core dll's have identical public surfaces.  And net472 automagically modiifies references to netstandard to load the specific net4X version of the api.
+
+## Assumptions
+For the purposes of this RFC we will assume that:
+
+# Scenarios
+Please contribute scenarios to the list below.
+
+1. General purpose library developer targetting usage on both NetCoreApp *, and net472+ frameworks uses F# to build an library.
+When the FSharp.Core targets netstandard 2.0  only then a nuget package with a single netstandard2.0 will work well, the developer will be able to build libraries that target all coreclr frameworks from netcoreapp2.0 up, and all desktop frameworks from net472 to net48.
+
+2. Application developer targetting windows development build a command line application  uses F# to build an application.
+The developer can target Windows using net472 and net48, or the coreclr with all netcoreapps from 2.0 up
+
+3. Application developer targetting linux / mac development build a command line application  uses F# to build an application.
+The developer can target Windows linux / mac or windows using the coreclr with all netcoreapps from 2.0 up
+
+4. Application developer targetting windows / linux / mac using net5 uses F# to build an app.
+The developer can target Windows linux / mac or windows using the coreclr with all net5 up
+
+5. General purpose library developer targetting usage on net45+ frameworks uses F# to build an library.
+netstandard2.0 is not well supported on versions of dotnet earlier than 4.7.2, in this circumstance the developer should reference the FSharp.Core nuget package version 4.7.3 or earlier to get the net45 version of FSharp.Core.  New language features that rely on an updated FSharp.Core will not work.
+
+5. Application developer targetting usage on net45+ frameworks uses F# to build an application.
+netstandard2.0 is not well supported on versions of dotnet earlier than 4.7.2, in this circumstance the developer should reference the FSharp.Core nuget package version 4.7.3 or earlier to get the net45 version of FSharp.Core.  New language features that rely on an updated FSharp.Core will not work.
+
+
+# Open Questions (General)
+
+# Alternatives
+1.  Continue targetting multiple frameworks
+Instead of building against a single netstandard2.0 framework we additionally target netstandard 2.1, net5 ... etc
+
+Response:
+Future frameworks are targetting release specific framework rids, I.e. netcoreapp3.0, netcoreapp3.1, net5 ... beyond.
+If we chose to target specific frameworks with FSharp.Core developerslibrary developers 281299 would be responsible for picking, developing and deploying rid specific FSharp.Cores.  Right now FSharp.Core doesn't need any rid specific code and so we will ship a single FSharp.Core with the widest reach possible.
```

Comment:
281299? :laughing:

Reply:
Exactly that number, not one more or one less.